src/core/default/commands/function/logs.py

```diff
@@ -43,9 +41,8 @@ def command(self, *args, **kwargs):
         full_function_name: str = kwargs.get("function_name")
         component_name = full_function_name.split('.')[0]
         function_name = full_function_name.split('.')[1]
-        watch_val = kwargs.get("watch")
-        tail_val = kwargs.get("tail")
-        number_val = kwargs.get("number")
+        # tail_val = kwargs.get("tail")
+        # number_val = kwargs.get("number")
```

Comment:
I am not gonna look too hard at these changes at the moment because they can be added as part of a larger improvement to the `logs` command. There should be a ticket for these improvements and are something we should tackle soon.